app/assets/stylesheets/fonts/glyphicons-halflings-regular.svg

```diff
@@ -0,0 +1,229 @@
+<?xml version="1.0" standalone="no"?>
```

Comment:
do we really need 3 different icons sets: `fontawesome`, `glyphicons`, `stroke 7`???